.travis.yml

```diff
@@ -12,10 +12,17 @@ before_install:
   - sleep 3
 
 script:
+  - pub run build_runner build
   - pub run dart_dev format --check
-  - pub run dart_dev analyze
-  - pub serve --port=59925 test &
-  - sleep 5s
-  - pub run test --concurrency=4 -p content-shell --reporter=expanded --pub-serve=59925 test/unit/generated_runner_test.dart
-  - pub run dart_dev coverage --no-html
-  - bash <(curl -s https://codecov.io/bash) -f coverage/coverage.lcov
+  # The analysis server can resolve elements from .dart_tool/build/, whereas the
+  # dartanalyzer cli cannot (yet). For this reason, tuneup is used because it
+  # leverages the analysis server. The dartanalyzer issue is being tracked here:
+  # https://github.com/dart-lang/sdk/issues/34098
+  - pub global activate tuneup
+  - pub global run tuneup check --ignore-infos
+  - pub run build_runner test -- --reporter=expanded
```

Comment:
fwiw, `pub run dart_dev test` would work here, too. ddev checks for the presence of `build` and `build_test` dependencies in the `pubspec.yaml` and will choose between `pub run build_runner test` and `pub run test` as necessary.